prombench/manifests/prombench/nodepools.yaml

```diff
@@ -1,5 +1,5 @@
 zone: {{ .ZONE }}
-projectid: {{ .PROJECT_ID }}
+projectid: {{ .GKE_PROJECT_ID }}
```

Comment:
We want to move this file aswell to the new file name, no? don't forget to update the same in the Makefile :)